gitlab/v4/objects/service_accounts.py

```diff
@@ -0,0 +1,21 @@
+from gitlab.base import RESTManager, RESTObject
+from gitlab.mixins import CreateMixin
+from gitlab.types import RequiredOptional
+
+__all__ = ["GroupServiceAccount", "GroupServiceAccountManager"]
+
+
+class GroupServiceAccount(RESTObject):
+    _id_attr = "id"
```

Comment:
We seem to still have the redundant id here:

```suggestion
    pass
```